test/e2e/ccm_helpers.go

```diff
@@ -88,6 +88,9 @@ func createService(svcName string, svcNamespace string, labels map[string]string
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: svcNamespace,
 			Name:      svcName,
+			Annotations: map[string]string{
+				"service.beta.kubernetes.io/oci-load-balancer-internal": "true",
```

Comment:
we are hitting public ip limits, hence changing the lb to he internal, should not matter for the test we are doing